### PR TITLE
[IAC-1786] IAC 0.181.0 Release Notes

### PR DESCRIPTION
**Pull request by richardblack-Harness**
### What
IACM's first release notes entry. Previous entries will be backfilled about 6-8 weeks after the first entry approval.

JIRA: [IAC-1786](https://harness.atlassian.net/browse/IAC-1786)

## PR lifecycle
PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

[IAC-1786]: https://harness.atlassian.net/browse/IAC-1786?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by douglas-j-bothwell — COMMENTED** [review]
Need a few tweaks. I'll be happy to review again.

File: release-notes/infrastucture-as-code-management.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Infrstructure as Code Management release notes
+sidebar_label: Infrstructure as Code Management
+date: 2024-04-18T10:00
```
[the remark this answers is not in the document]
Comment (richardblack-Harness) [reply release-notes/infrastucture-as-code-management.md:4]: Fair point

File: release-notes/infrastucture-as-code-management.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Infrstructure as Code Management release notes
+sidebar_label: Infrstructure as Code Management
```
[the remark this answers is not in the document]
Comment (richardblack-Harness) [reply release-notes/infrastucture-as-code-management.md:3]: Done

File: release-notes/infrastucture-as-code-management.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+:::
+
+## April 2024
+
+### Version: 0.181.0
+<!-- Released on: 04 April 2024 -->
+#### Layout & Design Refinement:
+  - **Enhanced Layout:** Increased spacing around store cards to prevent overlap of check marks and enhance readability and accessibility.
+  - **Sensitive Field Icons:** The eye/hide icon is now exclusively used with sensitive fields to improve privacy and clarity.
```
[the remark this answers is not in the document]
Comment (richardblack-Harness) [reply release-notes/infrastucture-as-code-management.md:29]: Ah yes, I see that from other modules now. Thanks

**Review by douglas-j-bothwell — APPROVED** [review]
LGTM

**Pull request merged**